src/components/itemidentifier/itemidentifier.js

```diff
@@ -261,64 +280,64 @@ define(['dialogHelper', 'loading', 'connectionManager', 'require', 'globalize',
 
         loading.show();
 
-        var options = {
+        const options = {
             ReplaceAllImages: page.querySelector('#chkIdentifyReplaceImages').checked
         };
 
-        var apiClient = getApiClient();
+        const apiClient = getApiClient();
 
         apiClient.ajax({
             type: 'POST',
-            url: apiClient.getUrl('Items/RemoteSearch/Apply/' + currentItem.Id, options),
+            url: apiClient.getUrl(`Items/RemoteSearch/Apply/${currentItem.Id}`, options),
             data: JSON.stringify(currentSearchResult),
             contentType: 'application/json'
 
-        }).then(function () {
+        }).then(() => {
 
             hasChanges = true;
             loading.hide();
 
             dialogHelper.close(page);
 
-        }, function () {
+        }, () => {
 
             loading.hide();
 
             dialogHelper.close(page);
         });
     }
 
-    function showIdentificationForm(page, item) {
+    function showIdentificationForm(page, {Id, Type}) {
```

Comment:
Same comment in most of these migrations about the `{}` expansion for parameters, but I'll wait to see what everyone else thinks.